### PR TITLE
update scala version, update deps

### PR DESCRIPTION
**Pull request by Terkwood**
Fixes https://nvd.nist.gov/vuln/detail/CVE-2022-36944

### CODE REVIEW

**Review by damianxu88 — APPROVED** [review]
LGTM

File: build.sbt [hunk 1]
```diff
@@ -22,7 +22,8 @@ val stubbornArtifact  = "com.krux"                  %% "stubborn"
 
 lazy val commonSettings = Seq(
   scalacOptions ++= Seq("-deprecation", "-feature", "-Xlint"), // , "-Xfatal-warnings"),
-  scalaVersion := "2.13.8",
+  // fix https://nvd.nist.gov/vuln/detail/CVE-2022-36944
```
Comment (realstraw) [inline build.sbt:25]: don't need to have a comment if you are not overriding anything. also can you also update other dependencies too?

File: build.sbt [hunk 2]
```diff
@@ -1,28 +1,33 @@
-val slickVersion = "3.3.3"
-val akkaVersion = "2.6.19"
-// make sure this is the same as the playWS's dependency
-val playJsonVersion = "2.9.2"
+val slickVersion = "3.4.1"
+val akkaVersion = "2.6.20"
```
Comment (realstraw) [inline build.sbt:2]: I believe this needs to be consistent with play's akka, otherwise it would give runtime errors

**Comment by Terkwood** [comment]
@realstraw whatcha think

File: build.sbt [hunk 3]
```diff
@@ -69,7 +71,8 @@ lazy val orchardWS = (project in file("orchard-ws")).
     buildInfoPackage := "com.salesforce.mce.orchard.ws",
     libraryDependencies ++= Seq(
       guice,
-      scalaPlusPlay
+      scalaPlusPlay,
+      akkaActor
```
Comment (realstraw) [inline build.sbt:75]: why do we need to add this to common?

Comment (Terkwood) [reply build.sbt]: trimmed

**Comment by Terkwood** [comment]
@realstraw ok one more time please

**Pull request merged**